source/tutorials/encrypt.txt

```diff
@@ -0,0 +1,300 @@
+.. _scala-encrypt:
+
+======================
+Client-Side Encryption
+======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, encrypt data, decrypt data
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Starting in v4.2, MongoDB supports client-side encryption. Client-side
+encryption allows administrators and developers to encrypt specific data
+fields in addition to providing other MongoDB encryption features.
+
+With field-level encryption, developers can encrypt fields on the client-side
+without any server-side configuration or directives. Client-side field
+level encryption supports workloads where applications must guarantee
+that unauthorized parties, including server administrators, cannot
+read the encrypted data.
+
+.. include:: /includes/obs-note.rst
+
+Installation
+------------
+
+The recommended way to get started using field-level encryption in
+your project is by using a dependency management system. Field-level
+encryption requires additional packages in addition to the
+driver.
+
+.. include:: /includes/install-note.rst
+
+libmongocrypt
+~~~~~~~~~~~~~
+
+There is a separate JAR file containing ``libmongocrypt`` bindings.
+
+.. tabs::
+
+   .. tab:: Maven
+      :tabid: maven-installation
+
+      If you are using `Maven <https://maven.apache.org/>`__ to manage your
+      packages, add the following entry to your ``pom.xml`` dependencies list:
+
+      .. code-block:: xml
+
+         <dependencies>
+             <dependency>
+                 <groupId>org.mongodb</groupId>
+                 <artifactId>mongodb-crypt</artifactId>
+                 <version>1.2.1</version>
+             </dependency>
+         </dependencies>
+
+   .. tab:: Gradle
+      :tabid: gradle-installation
+
+      If you are using `Gradle <https://gradle.org/>`__ to manage your
+      packages, add the following entry to your dependencies list:
+      
+      .. code-block:: scala
+
+         dependencies {
+             implementation 'org.mongodb:mongodb-crypt:1.2.1'
+         }
+
+mongocryptd Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``libmongocrypt`` bindings require the ``mongocryptd`` daemon/process to be
+running. A specific daemon/process URI can be configured in the
+``AutoEncryptionSettings`` class by setting ``mongocryptdURI`` in the
+``extraOptions`` setting.
+
+Examples
```

Comment:
Suggestion:

No change needed, but I think there are four code examples here, but only two are identified with headers. I think the following headers would work for the first two examples:

1) Auto Encryption and Decryption
2) Specify a Data Encryption Key and Encrypted Fields